src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/AzureRM.Resources.ps1

```diff
@@ -0,0 +1,158 @@
+﻿function Get-AzureRmResourceGroup
```

Comment:
nit: I assume VS doesn't do a good job on aligning PS, but we should maintain that.